content/filterx/function-reference.md

```diff
@@ -131,11 +159,25 @@ The `value_separator` option must be a single character, the `pair_separator` ca
 format_kv(<input-dictionary>, value_separator=":", pair_separator=";")
 ```
 
-## format_json {#format-json}
+## format_leef {#format-leef}
 
-Formats any value into a raw JSON string.
+Available in {{< product >}} 4.13 and later.
 
-Usage: `format_json($data)`
+Formats a dictionary into the [Log Event Extended Format (LEEF)](https://www.ibm.com/docs/en/SS42VS_DSM/pdf/b_Leef_format_guide.pdf). Both LEEF versions (1.0 and 2.0) are supported.
+
+Usage: `${MESSAGE} = format_leef(my_dictionary);`
+
+For example:
+
+```json
+my_dictionary = {"version":"1.0","vendor":"Microsoft","product_name":"MSExchange","product_version":"4.0 SP1","event_id":"15345","src":"192.0.2.0","dst":"172.50.123.1","sev":"5cat=anomaly","srcPort":"81","dstPort":"21","usrName":"joe.black"};
+```
+
+Becomes:
+
+```shell
+LEEF:1.0|Microsoft|MSExchange|4.0 SP1|15345|src=192.0.2.0 dst=172.50.123.1 sev=5cat=anomaly srcPort=81 dstPort=21 usrName=joe.black
```

Comment:
LEEF's extensions(src, dst, ...) are tab separated, not sure if these are that or space separated.